### PR TITLE
fix(gear): Fix gear handle behavior on A380

### PR DESCRIPTION
**Pull request by crocket63**
Fixes #[issue_no]

## Summary of Changes

Fix auto gear retract when getting of the ground

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<ModelBehaviors>
+    <!--
+    Template for gear lever movement
+    -->
+    <Template Name="FBW_HANDLING_Gear_Lever_Template">
+        <DefaultTemplateParameters>
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml:6]: Why using the legacy parameter format for new templates?

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            </DRAG_CODE>
+        </DefaultTemplateParameters>
+
+        <Component ID="#NODE_ID#" Node="#NODE_ID#">
+            <UseTemplate Name="ASOBO_GT_Lever_DraggingYAxis_Code">
+                <ANIM_LENGTH>3</ANIM_LENGTH>
+                <ANIM_CODE>
+                        (L:A32NX_GEAR_HANDLE_POSITION, Percent) 3 *
+                </ANIM_CODE>
+                <UP_CODE>  (&gt;K:GEAR_UP)</UP_CODE>
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml:39]: Does this actually do anything when you're overriding the LEFT_xx_CODE with your template params?

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                } els{
+                    (&gt;K:GEAR_DOWN)
+                }
+            </LEFT_SINGLE_CODE>
+            <LEFT_RELEASE_CODE>
+                (L:A32NX_GEAR_LEVER_LOCKED, Boolean) 1 ==  if{
+                    (&gt;K:GEAR_DOWN)
+                }
+            </LEFT_RELEASE_CODE>
+            <DRAG_CODE>
```
Comment (tracernz) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml:28]: Doesn't this disable dragging the lever with the mouse?

**Comment by tracernz** [comment]
Would be better to use IEs for this stuff.

**Comment by Maximilian-Reuter** [comment]
> > Would be better to use IEs for this stuff.
> 
> What is IE ? Yet point of this PR is just to copy what's done in 320 code, not update it. If we can make it better and simpler I'm all down for it though, for both planes.

Tracer likes to use abbreviations, not realising that not everyone works with msfs lingo everyday :D IE are InputEvents
https://docs.flightsimulator.com/html/Samples_And_Tutorials/Tutorials/Creating_Interactions_Using_InputEvents.htm

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml (hunk 1 above)
Comment (tracernz) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/generic/gear.xml:6]: Okay, let's keep it and refactor the templates later I guess.

**Pull request merged**